### ISSUE TITLE
Crashing the osjs when uploading a file in osjs mountpoint

### ISSUE THREAD

**Issue by maryam4s26** [os-js/OS.js#840]
hi.
when I upgrade node version to 16 and upload the file in osjs mountpoint, osjs crashes. How can I fix this?

@andersevenrud 
@ajmeese7

![Screenshot from 2023-12-24 18-07-19](https://github.com/os-js/OS.js/assets/65979871/97ef12d2-5960-4d65-acc4-a423d9a9e676)

**Comment by maryam4s26** [comment]
I think we should handle the error in this way so that it does not crash:
https://github.com/os-js/osjs-server/blob/master/src/utils/vfs.js#L71

Change from code 1 to 2:

1. 

```
return isStream
    ? req.files.upload
    : fs.createReadStream(req.files.upload.path);
```

2. 

```
return isStream
    ? req.files.upload
    : fs.createReadStream(req.files.upload.path)
        .on('error', function(err) {})
```

**Comment by andersevenrud** [comment]
Thanks for reporting! 👍 

I'm away because of the holidays, so I cannot look into this myself at the moment.

Feel free to submit a PR to the osjs-server repository if you have a working solution. This is probably also something that could use a unit test to verify the problem and the solution.

**Comment by andersevenrud** [comment]
Just published `@osjs/server@3.4.0` that includes the fix.
